python/fledge/services/core/api/asset_tracker.py

```diff
@@ -128,4 +132,8 @@ async def deprecate_asset_track_entry(request: web.Request) -> web.Response:
         msg = str(ex)
         raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
     else:
+        audit = AuditLogger(storage_client)
+        audit_details = {'asset': asset_name, 'service': svc_name, 'event' : event_name}
+        await audit.information('ASTDP', audit_details)
+        _logger.info("Asset '{}' has been deprecated".format(asset_name))
```

Comment:
Move this to try block after if response != 'updated': Otherwise this shall create audit entry even on failure/ bad requests/already deprecated stuff.